core/tests/protractor_desktop/additionalEditorAndPlayerFeatures.js

```diff
@@ -70,6 +70,23 @@ describe('Full exploration editor', function() {
     }
   );
 
+  it('should report an exploration', function() {
```

Comment:
Change to `should report an exploration to moderators`

Reply:
Done